hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/PerformanceEvaluation.java

```diff
@@ -2276,7 +2319,38 @@ boolean testRow(final int i, final long startTime) throws IOException {
       return true;
     }
   }
+  static class MetaWriteTest extends Test {
+
+    MetaWriteTest(Connection con, TestOptions options, Status status) {
+      super(con, options, status);
+    }
+
+    @Override
+    void onStartup() throws IOException {
+    }
 
+    @Override
+    void onTakedown() throws IOException {
+    }
+    @Override
+    boolean testRow(final int i, final long startTime) throws IOException {
+      List<RegionInfo> regionInfos = new ArrayList<RegionInfo>();
+
+      for (int index = 0; index < i; index++) {
+        regionInfos.add(RegionInfoBuilder.newBuilder(TableName.valueOf("hbase:meta"))
```

Comment:
This will be a bit confusing that we add "hbase:meta" to hbase:meta?

Reply:
Thanks. changed to the default test table.